packages/lambda-api-tracker/src/tile-request.ts

```diff
@@ -0,0 +1,58 @@
+import { Projection } from '@basemaps/geo';
+import {
+    Const,
+    HttpHeader,
+    LambdaHttpResponse,
+    LambdaHttpResponseCloudFront,
+    LambdaHttpResponseCloudFrontRequest,
+    ReqInfo,
+    tileFromPath,
+} from '@basemaps/lambda-shared';
+import { queryStringExtractor } from './query';
+import { ReqInfoCloudFront } from './req-info-cloudfront';
+import { ValidateRequest } from './validate';
+
+const projection = new Projection(256);
+
+/**
+ * Validate a CloudFront request has a valid API key and is not abusing the system
+ */
+export default async (info: ReqInfo): Promise<LambdaHttpResponse> => {
+    // Expose some debugging metrics so we can plot tile requests onto a map
+    const xyzData = tileFromPath(info.rest);
+    if (xyzData == null) return new LambdaHttpResponseCloudFront(404, 'Not Found');
```

Comment:
Other URLS need to be supported get `/WMTSCapabilites.xml` if you 404 here they wont be able to be accessed.

you need to return the `request` to allow the request to go forward

Reply:
Yep will do that as part of the WMTS ticket

Reply:
Ok lets review this after the WMTS work is done.